src/main/java/racingcar/global/constants/SymbolType.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.global.constants;
+
+public enum SymbolType {
+    CAR_NAME_DELIMITER(","),
+    RESULT_POSITION("-"),
+    RESULT_TOKEN(" : "),
+    FINAL_RESULT_TOKEN(", "),
```

Comment:
```suggestion
    RESULT_POSITION_UNIT("-"),
    RESULT_DELIMITER(" : "),
    FINAL_RESULT_DELIMITER(", "),
```
TOKEN은 DELIMITER로 분리된 것들인 것 같습니다.

Reply:
네이밍이 명쾌하십니다 ... 
더 명확한 단어를 사용하는 것에 한 수 배워갑니다!